### PR TITLE
fix(ecosystem): Resize issue linking headers

### PR DESCRIPTION
**Pull request by scttcper**
resizes the headers for integrations 
image

### CODE REVIEW

File: static/app/components/externalIssues/abstractExternalIssueForm.tsx [hunk 1]
```diff
@@ -337,7 +337,9 @@ export default class AbstractExternalIssueForm<
 
     return (
       <Fragment>
-        <Header closeButton>{this.getTitle()}</Header>
+        <Header closeButton>
+          <h4>{this.getTitle()}</h4>
```
Comment (evanpurkhiser) [inline static/app/components/externalIssues/abstractExternalIssueForm.tsx:341]: you know. When i refactored all the headers, I tried to REMOVE all these h4s, in hopes of just changing the standard header size.

How hard do you think that would be?

Comment (scttcper) [reply static/app/components/externalIssues/abstractExternalIssueForm.tsx:341]: you'd have to fight @robinrendle

Comment (priscilawebdev) [reply static/app/components/externalIssues/abstractExternalIssueForm.tsx:341]: if we follow this pattern of always having `<h4>` in the header of modals, we must create a reusable component for it

**Review by priscilawebdev — APPROVED** [review]
Thanks for working on this Scott! lgtm

**Pull request merged**